__tests__/doctors/[id].test.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { render, screen, waitFor } from '@testing-library/react';
 import DoctorDetail from '../../pages/doctors/[id]';
-import { mockDoctorData } from '@/mocks/doctor';
+import { mockDoctorData } from '@/__tests__/mocks/doctor';
```

Comment:
I think pr #52 take care of this import issue. If it won't be solved by Zeras Pr I will be reviewing this pr.